caddytls/certificates.go

```diff
@@ -149,14 +164,26 @@ func makeCertificateFromDisk(certFile, keyFile string) (Certificate, error) {
 	if err != nil {
 		return Certificate{}, err
 	}
-	return makeCertificate(certPEMBlock, keyPEMBlock)
+	// XXX: Get a Config and pass the real CTLogURLs
+	return makeCertificate(certPEMBlock, keyPEMBlock, []string{})
+}
+
+var x509SCTOid = asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 11129, 2, 4, 2}
```

Comment:
I think I know what this means, but maybe put a comment on this, it's a bit obscure. :)